IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/settings/wolfcrypt_benchmark_Debug.jlink

```diff
@@ -0,0 +1,39 @@
+[BREAKPOINTS]
```

Comment:
able to cleanup all the files noted by David except jlink. This contained the custom settings we needed to get "printf" re-directed to the Terminal/IO window in IAR. Without the proper kHZ settings we added, landis gyr would not be able to see the output of the tests.